### ISSUE TITLE
Issues in build

### ISSUE THREAD

**Issue by robertop87** [virtua-tech/polymer-typescript-starter-kit#21]
I just run **polymer serve build**, and I open the localhost:8081, nothing is shown and I got the next error on my devtools console
Uncaught TypeError: Cannot destructure property `customElement` of 'undefined' or 'null'.

**Comment by samcarecho** [comment]
Have you run "npm run build" before "polymer serve"?

Try the following sequence of commands:

```
npm run build
polymer build
polymer serve build/[build_name]
```

[build_name] can be the name of any of the directories under the build directory.

**Comment by kito99** [comment]
@robertop87 you should just be able to run `npm run build` and then `polymer serve`.

**Comment by robertop87** [comment]
Thanks for the support at the end it was fixed updating the npm to 5.6.0.
The previous version was 4.9.1.

Not sure how it was related but now the issue gone.
